charts/enterprise/authelia/questions.yaml

```diff
@@ -982,7 +1007,9 @@ questions:
                                         description: "two_factor"
                                 - variable: consent_mode
                                   label: "Consent Mode"
-                                  description: "Configures the consent mode. This can be set to auto (default), explicit (consent required every time) or implicit (automatically assumes consent for every authorization, never asking the user if they wish to give consent.)"
+                                  description: |
+                                    "Configures the consent mode. This can be set to auto (default), explicit (consent required every time) or 
+                                    implicit (automatically assumes consent for every authorization, never asking the user if they wish to give consent.)"
```

Comment:
```suggestion
                                  description: |
                                    Configures the consent mode. This can be set to auto (default), explicit (consent required every time) or 
                                    implicit (automatically assumes consent for every authorization, never asking the user if they wish to give consent.)
```